src/cpp/types/endpoint/Endpoint.cpp

```diff
@@ -96,5 +96,14 @@ Endpoint& Endpoint::operator =(
     return *this;
 }
 
+std::ostream& operator <<(
+        std::ostream& os,
+        const Endpoint& endpoint)
+{
+    os << "{" << endpoint.guid_ << ";" << endpoint.topic_ << ";" << endpoint.qos_ << ";" << endpoint.kind_ << ";"
```

Comment:
I'd include the meaning of each field as it was a dictionary {key: value, key: value,...}.
This applies to all `<<` that return a dictionary

Reply:
I think this would make the entity more difficult to read. @juanlofer-eprosima what do you think about it?

Reply:
I would use `std::endl` as separator if it's possible, and then adding the keys would be good.